examples/dt/uni-alpha/control-plane/service-values.yaml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = lvm
+
+      [lvm]
+      image_volume_cache_enabled = false
+      volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
+      volume_group = cinder-volumes
+      target_protocol = iscsi
+      target_helper = lioadm
+      volume_backend_name = lvm_iscsi
+
+  cinderBackup:
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.swift
+    replicas: 1
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+
+      [glance_store]
+      default_backend = default_backend
+
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+
+  ironic:
+    enabled: true
+
+  swift:
+    enabled: true
+
+  octavia:
+    enabled: true
+
+  telemetry:
+    enabled: true
+    autoscaling:
+      enabled: true
+    ceilometer:
+      enabled: true
+
+  ovn:
+    enabled: false
```

Comment:
Discussed this further with team, instead of disabling ovncontroller we want to have ovnController pods in zone-1 az and networker-nodes ovn-controller on zone-2 az. So here we need to configure instead of above like:-
ovn:
  template:
    ovnController:
      external-ids:
        availability-zones: ["zone-1"]

Reply:
On the controller, the rendered output is
```
  ovn:
    template:
      ovnController:
        external-ids:
          availability-zones:
          - zones-1
        networkAttachment: tenant
        nicMappings:
          datacentre: ospbr
      ovnDBCluster:
        ovndbcluster-nb:
          dbType: NB
          networkAttachment: internalapi
          storageRequest: 10G
        ovndbcluster-sb:
          dbType: SB
          networkAttachment: internalapi
          storageRequest: 10G
      ovnNorthd:
        networkAttachment: internalapi
```

Hope this is on the expected lines.